### PR TITLE
cargo*: add Chinese translation

### PR DESCRIPTION
**Pull request by Beriholic**
- [x ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x ] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Beriholic** [comment]
My fault.

File: pages.zh/common/cargo-build.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- 以 release 模式构建，并进行优化：
+
+`cargo build --release`
+
+- 要求 `Cargo.lock` 文件为最新版本：
+
+`cargo build --locked`
+
+- 构建工作区中的所有包
```
Comment (blueskyson) [inline pages.zh/common/cargo-build.md:18]:
```suggestion
- 构建工作区中的所有包：
```

File: pages.zh/common/cargo-vendor.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# cargo vendor
+
+> 将项目的所有依赖项存储到指定目录中 (默认为`vendor`)
```
Comment (blueskyson) [inline pages.zh/common/cargo-vendor.md:3]:
```suggestion
> 将项目的所有依赖项存储到指定目录中（默认为 `vendor`）。
```

File: pages.zh/common/cargo.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# cargo
+
+> 管理 Rust 项目及其模块依赖项（crates）。
+> 一些子命令，如`build`，具有自己的用法文档。
```
Comment (blueskyson) [inline pages.zh/common/cargo.md:4]:
```suggestion
> 一些子命令，如 `build`，具有自己的用法文档。
```

File: pages.zh/common/cargo-bench.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# cargo bench
+
+> 编译并执行基准测试
+> 更多信息: <https://doc.rust-lang.org/cargo/commands/cargo-bench.html>.
```
Comment (blueskyson) [inline pages.zh/common/cargo-bench.md:4]: Please fix all the other files as well.
```suggestion
> 编译并执行基准测试。
> 更多信息：<https://doc.rust-lang.org/cargo/commands/cargo-bench.html>.
```

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

**Pull request merged**